webapp/portlet/src/main/webapp/vue-apps/common/js/ActivityService.js

```diff
@@ -91,6 +91,8 @@ export function shareActivity(activityId, message, templateParams, spaces) {
     method: 'POST',
     body: JSON.stringify({
       title: message,
+      body: title,
+      summary: summary,
```

Comment:
we may send the ID of the original activity and then get its title & summary from backend with no need to send them with the shared activity inofmration

Reply:
this tow information title and summary of activity shared is not recovred in [https://github.com/Meeds-io/social/blob/aa6c52bce981d1094fcf64fe7df19fe56943750e/component/core/src/main/java/org/exoplatform/social/core/jpa/storage/entity/ActivityEntity.java](url)

Reply:
title i can get it from model but summary is not stored in ActivityEntity , i need to add in Model!